README.md

```diff
@@ -245,6 +245,7 @@ Here is a full list of options:
 | `--background`               | `#fff`                                                                                                                                    | `#222`         | Background of the entire `<emoji-picker>`                                                            |
 | `--border-color`             | `#e0e0e0`                                                                                                                                 | `#444`         |                                                                                                      |
 | `--border-size`              | `1px`                                                                                                                                     |                | Width of border used in most of the picker                                                           |
+| `--border-radius`            | `0rem`                                                                                                                                    |                | Border radius of the picker menu.                                                                    |
```

Comment:
If you run `pnpm build` then this will be auto-generated from the `*.scss` file. It will also update the custom elements manifest.